hoot-services/src/main/java/hoot/services/controllers/conflation/ConflateCommand.java

```diff
@@ -72,6 +72,10 @@
         options.add("hootapi.db.writer.overwrite.map=true");
         options.add("writer.text.status=true");
         options.add("hootapi.db.writer.job.id=" + jobId);
+        options.add("hootapi.db.writer.remap.ids=false");
```

Comment:
Conflate uses it if you are writing back to the HootAPI database, which in this case we are.  Any hoot operation that writes to the database will be affected by this setting.

Reply:
Is this param key/value combo a candidate for becoming a default?

Reply:
Yes it will with the negative IDs in the database work.
https://github.com/ngageoint/hootenanny/pull/3020/files#diff-321008ef2555f92f76799b5cf24ad765